pulsar-functions/instance/src/main/python/python_instance_main.py

```diff
@@ -113,7 +113,7 @@ def main():
       zpfile = zipfile.ZipFile(str(args.py), 'r')
       zpfile.extractall(os.path.dirname(str(args.py)))
     sys.path.insert(0, os.path.dirname(str(args.py)))
-  elif os.path.splitext(str(args.py))[1] == '.zip':
+  elif zipfile.is_zipfile(str(args.py)):
```

Comment:
can we add a test for this?

Reply:
Yeah, I'm new to contributing so any direction on how to go about doing that would be helpful, but I'll dive in see if I can figure out how to create some tests around this.

Reply:
You can take a look at `test_python_instance.py`.

Reply:
@zymap I think we don't test 3rd-party dependency for their own functionality :)